tests/dummy/app/query-params/route.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import route from 'ember-redux/route';
+
+var queryParams = {
+  foo: {
+    refreshModel: false
+  }
+};
+
+function afterModel(dispatch, model, transition) {
+  if (Ember.isPresent(transition)) {
+    dispatch({
+      queryParams: Object.keys(this.get('queryParams')),
+      type: 'SET_QUERY_PARAMS'
+    });
+  }
+}
+
+var QueryParamsRoute = Ember.Route.extend();
```

Comment:
you shouldn't need this line - if you only write functions like this our `route` helper will provide a full `Ember.Route` for you (like ember generates controllers when you don't need to override anything)